lib/wwwtxt.rb

```diff
@@ -1,5 +1,7 @@
 require "wwwtxt/version"
+require "wwwtxt/page"
+require "wwwtxt/plain_text_renderer"
+require "wwwtxt/url_fetcher"
```

Comment:
Aye. For now it can be exactly as you're doing, just every file 'hardcoded'. When the project gets larger you can write whatever to do it dynamically.